package.json

```diff
@@ -4,7 +4,7 @@
   "version": "0.0.0",
   "repository": "git@github.com:backstage/community-plugins.git",
   "engines": {
-    "node": "18 || 20"
+    "node": "22"
```

Comment:
Does this change suggest the removal of support for node 18 and 20 for the community plugins? It feels to me like 
it should be supporting a range of versions making it more flexible and allow projects to migrate?

Reply:
I agree with the above concern. By blanket updating we could unnecessarily break users who happen to have enginesStrict enabled. 

IMO these fields should be communication from the workspace/plugin owner about compatibility rather than a blanket change applied across the repository.

I can take a look at breaking out the engines changes and the Node.js 22 updates.